### ISSUE TITLE
Set TF_IN_AUTOMATION environment variable

### ISSUE THREAD

**Issue by findkim** [hashicorp/consul-terraform-sync#280]
#### Description
Setting the environment variable [`TF_IN_AUTOMATION`](https://www.terraform.io/docs/cli/config/environment-variables.html#tf_in_automation) may be a nice enhancement when CTS transitions into daemon mode, where Terraform is then run in automation. This would toggle cosmetic changes for Terraform command outputs to avoid suggesting commands to run next as if a user is actively engaging with Terraform.

#### Additional context
What do the output changes look like?

**Comment by wilkermichael** [comment]
I started looking into this, and this is what I found:

- CTS is using the terraform-exec Go module to create a new terraform client in client/terraform_cli.go 
	`tf, err := tfexec.NewTerraform(config.WorkingDir, tfPath)`
- As per the comments in the terraform-exec module, it overrides a number of terraform environment variables when using it's terraform client, including `TF_IN_AUTOMATION`:
https://github.com/hashicorp/terraform-exec/blob/43e54be55ebc3ce07d381660dcafcb45f567e67c/tfexec/terraform.go#L31-L42
- We see that the terraform-exec is setting the environment variable `TF_IN_AUTOMATION` to `1`
https://github.com/hashicorp/terraform-exec/blob/43e54be55ebc3ce07d381660dcafcb45f567e67c/tfexec/cmd.go#L22
https://github.com/hashicorp/terraform-exec/blob/43e54be55ebc3ce07d381660dcafcb45f567e67c/tfexec/cmd.go#L156-L157

So in summary, it looks like CTS, by virtue of the terraform-exec module, is already setting TF_IN_AUTOMATION to 1. As per the [documentation](https://www.terraform.io/docs/cli/config/environment-variables.html#tf_in_automation) Terraform should be adjusting its output to avoid suggesting specific commands to run next.

@findkim let me know if the above breakdown is what we were looking for in this issue, or if there are other things I should look into as well.

**Comment by findkim** [comment]
🕵️‍♂️ oh nice that's great to see it's already handled. It does make sense that the library would make that assumption and set the variable instead of having CTS manage that. Thanks for write up, it was clear to follow!
